virtcontainers/qemu.go

```diff
@@ -621,16 +627,24 @@ func (q *qemu) startSandbox(timeout int) error {
 		timeStart := time.Now()
 		go func() {
 			scanner := bufio.NewScanner(stderr)
+			var sent bool
 			for scanner.Scan() {
-				if strings.Contains(scanner.Text(), "Waiting for vhost-user socket connection...") {
+				if q.config.Debug {
+					q.Logger().WithField("source", "virtiofsd").Debug(scanner.Text())
+				}
+				if !sent && strings.Contains(scanner.Text(), "Waiting for QEMU socket connection...") {
```

Comment:
s/QEMU/vhost-user/

Reply:
Nice catch! Yeah, I need to update my virtiofsd as well.

Reply:
fixed. ptal.